frontend/src/router/auth.ts

```diff
@@ -29,8 +29,9 @@ const authRoutes: RouteRecordRaw[] = [
       {
         path: "admin",
         name: AUTH_SIGNIN_ADMIN_MODULE,
-        meta: { title: () => t("common.sign-in-as-admin") },
-        component: () => import("@/views/auth/SigninAdmin.vue"),
+        redirect: {
+          name: AUTH_SIGNIN_MODULE,
```

Comment:
It seems that once email-password login is disabled, the frontend will no longer show the signin form.